### DISCUSSION TITLE
为什么 OBS除了用“显示器采集”之外都采集不到PCL

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#2777] [提问] [answered]
使用“窗口采集”没有PCL应用
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/6415a431-1270-4d60-926a-4b9f2a827b10)
使用“游戏采集”采了个寂寞
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/dca8e363-5147-4244-ba47-d30bbb26d582)

**Comment by MoYuan-CN** [comment]
好像 是因为 窗口是 无标题化（`WindowStyle="None"`）的？
不过我在测试的时候发现能采集鼠标的移动 不过是下面这样的
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/56914dfd-3988-4b67-9824-22117764fb26)
~~我的无标题化App也没法采集 悲~~

**Comment by SqrtTan** [comment]
我也发现有这问题……（有大碍吗但是）

**Comment by WForst-Breeze** [comment, marked as answer]
如图设置即可
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/3f432358-fb6d-48c3-b657-61f3fc212443)

**Reply by MoYuan-CN** [reply]
啊哈 好了

**Reply by kong-que-dnf123** [reply]
谢了awa
